.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.18.0.cjs"
```

Comment:
this silenced an error I was seeing when adding dependencies to this package

```
error An unexpected error occurred: "expected workspace package to exist for \"@babel/template\"".
```

Resolved by running `yarn policies set-version 1.18` which generated these files. See https://github.com/yarnpkg/yarn/issues/7807 for more